mmvb_backend/case_synthesizer/api/schemas.py

```diff
@@ -0,0 +1,48 @@
+import warnings
+
+from rest_framework.exceptions import APIException
+from rest_framework.status import HTTP_201_CREATED
+
+from common.utils import CamelCaseAutoSchema
+
+
+class CaseSynthesizerSchema(CamelCaseAutoSchema):
+    def get_response_serializer(self, path, method):
+        view = self.view
+        try:
+            return view.get_response_serializer()
+        except APIException:
+            warnings.warn(
+                "{}.get_custom_serializer() raised an exception during "
+                "schema generation. Serializer fields will not be "
+                "generated for {} {}.".format(
+                    view.__class__.__name__, method, path
+                )
+            )
+
+    def get_responses(self, path, method):
+        self.response_media_types = self.map_renderers(path, method)
+
+        serializer = self.get_response_serializer(path, method)
+        item_schema = self._get_reference(serializer)
+        response_schema = {
+            "type": "array",
+            "items": item_schema,
+        }
+
+        paginator = self.get_paginator()
```

Comment:
Maybe I misunderstand how the paginator works, but (in my understanding) creating a case set is *not idempotent*, so if I call it a second time to get the next page... wouldn't I create (accidentally) create a second case set?
More generally: should the generator even return the generated data, or simply the URI where to fetch it?

Reply:
Paginators would serve the purpose of splitting long responses into multiple pages and you might be right about the possible behaviour in this case. But for this specific implementation we're just keeping part of the original implementation that we have overridden.

Since there is currently no paginator configured/implemented for that, having that bit of code should be harmless, but leaving it explicit that we could have pagination implemented if we want to (and do it our way, so avoiding to create another set of cases).

In regards to returning a list of URIs to fetch the generated cases it is feasible if you think this behaviour would be better for the frontend.